os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -440,6 +440,43 @@ static void _up_assert(int errorcode)
 }
 
 #ifdef CONFIG_BINMGR_RECOVERY
+static void recovery_unblock_task(struct tcb_s *tcb)
+{
+	struct tcb_s *ntcb;
+
+	dq_rem((FAR dq_entry_t *)tcb, (dq_queue_t *)g_tasklisttable[tcb->task_state].list);
```

Comment:
How about using definition for this line?
```
#define REM_CURRENT_LIST(tcb)  dq_rem(......)

static void recovery_unblock_task(struct tcb_s *tcb)
{
	REM_CURR_LIST(tcb);
	ADD_READYTORUN_LIST(tcb);
	...
```
It is easier to understand what it does.